src/test/scala/inox/evaluators/EvaluatorSuite.scala

```diff
@@ -118,20 +161,59 @@ class EvaluatorSuite extends FunSuite {
     eval(e, LessThan(IntegerLiteral(4), IntegerLiteral(7)))       === BooleanLiteral(true)
   }
 
-  test("Int Modulo and Remainder") {
+  test("BitVector Comparisons") {
+    val e = evaluator(ctx)
+
+    eval(e, GreaterEquals(Int8Literal(7), Int8Literal(4)))  === BooleanLiteral(true)
+    eval(e, GreaterThan(Int8Literal(7), Int8Literal(7)))    === BooleanLiteral(false)
+    eval(e, LessEquals(Int8Literal(7), Int8Literal(7)))     === BooleanLiteral(true)
+    eval(e, LessThan(Int8Literal(4), Int8Literal(7)))       === BooleanLiteral(true)
+
+    eval(e, GreaterEquals(Int32Literal(7), Int32Literal(4)))  === BooleanLiteral(true)
+    eval(e, GreaterEquals(Int32Literal(7), Int32Literal(7)))  === BooleanLiteral(true)
+    eval(e, GreaterEquals(Int32Literal(4), Int32Literal(7)))  === BooleanLiteral(false)
+
+    eval(e, GreaterThan(Int32Literal(7), Int32Literal(4)))    === BooleanLiteral(true)
+    eval(e, GreaterThan(Int32Literal(7), Int32Literal(7)))    === BooleanLiteral(false)
+    eval(e, GreaterThan(Int32Literal(4), Int32Literal(7)))    === BooleanLiteral(false)
+
+    eval(e, LessEquals(Int32Literal(7), Int32Literal(4)))     === BooleanLiteral(false)
+    eval(e, LessEquals(Int32Literal(7), Int32Literal(7)))     === BooleanLiteral(true)
+    eval(e, LessEquals(Int32Literal(4), Int32Literal(7)))     === BooleanLiteral(true)
+
+    eval(e, LessThan(Int32Literal(7), Int32Literal(4)))       === BooleanLiteral(false)
+    eval(e, LessThan(Int32Literal(7), Int32Literal(7)))       === BooleanLiteral(false)
+    eval(e, LessThan(Int32Literal(4), Int32Literal(7)))       === BooleanLiteral(true)
+
+    def bvl(x: BigInt) = BVLiteral(x, 13)
+    eval(e, GreaterEquals(bvl(7), bvl(4)))  === BooleanLiteral(true)
+    eval(e, GreaterThan(bvl(7), bvl(7)))    === BooleanLiteral(false)
+    eval(e, LessEquals(bvl(7), bvl(7)))     === BooleanLiteral(true)
+    eval(e, LessThan(bvl(4), bvl(7)))       === BooleanLiteral(true)
+  }
+
+  test("BitVector Division and Remainder") {
+    // FIXME is it intentional that Modulo are not tested?
```

Comment:
Was it intentional? Note that the test was initially called *Int Modulo and Remainder*.